tests/common/protocol/test_hostplugin.py

```diff
@@ -365,8 +365,7 @@ def test_fallback_channel_failure(self, patch_put, patch_upload, _):
             # ensure host plugin is not set as default
             self.assertFalse(wire.HostPluginProtocol.is_default_channel)
 
-    @patch("azurelinuxagent.common.event.add_event")
```

Comment:
I reduced the scope of this mock to the call to upload_status_blob(), otherwise it ends up capturing extra telemetry